kod/object/item/passitem/necklace/shadowam.kod

```diff
@@ -215,6 +219,27 @@ messages:
       return -viLight_change;
    }
 
+   CanIdentify()
+   {
+      return TRUE;
+   }
+
+   RevealHiddenAttributes()
+   {
+      if pbIdentified = FALSE
+      {
+         pbIdentified = TRUE;
+         Send(poOwner, @SomethingChanged, #what=self);
+      }
+
+      return pbIdentified;
```

Comment:
pbIdentified is always TRUE here

Reply:
Fixed to only return TRUE if `pbIdentified` was changed from FALSE to TRUE, otherwise returns FALSE.